public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -370,6 +372,17 @@ export class DashboardScenePageStateManagerV2 extends DashboardScenePageStateMan
 > {
   private dashboardLoader = new DashboardLoaderSrvV2();
 
+  async loadSnapshotScene(slug: string): Promise<DashboardScene> {
+    const rsp = await this.dashboardLoader.loadDashboard('snapshot', slug, '');
```

Comment:
How about introducing `loadSnapshot` method on the dashboard loader so that we don't rely on this weird signature with dashboard `type` (`loadDashboard`)? The implementation could live in the `DashboardLoaderSrvBase` as its exactly the same across v1/v2, and here we would simply call `loadSnapshot` by slug.